ompi/Makefile.am

```diff
@@ -128,9 +129,9 @@ DIST_SUBDIRS = \
 
 # Build the main MPI library
 
-lib_LTLIBRARIES = libmpi.la
-libmpi_la_SOURCES =
-libmpi_la_LIBADD = \
+lib_LTLIBRARIES = lib@OMPI_LIB_PREFIX@.la
```

Comment:
It's a little odd to call this value a "prefix", when, in fact, it's the entire library name.  I.e., it's:

```
lib@MPI_LIBRARY_NAME@.la
```

not

```
lib@OMPI_LIB_PREFIX@mpi.la
```

The latter would follow suit for what was done with the other libraries.  I.e., you could set the prefix to be `josh_`, and then it would be `libjosh_mpi` (etc.).

Is there a reason you want it to be the entire library name, rather than a prefix?